examples/hlb_cifar10.py

```diff
@@ -24,12 +27,12 @@ def __init__(self, channels_in, channels_out, short, se=True):
 
   def __call__(self, x):
     x = self.conv[0](x).max_pool2d(2)
-    x = self.norm[0](x).relu()
+    x = self.norm[0](x).clip(1e-15, 1e15).gelu()
     if self.short: return x
     residual = x
-    mult = self.se2(self.se1(residual.mean((2,3))).relu()).sigmoid().reshape(x.shape[0], x.shape[1], 1, 1) if self.se else 1.0
-    x = self.norm[1](self.conv[1](x)).relu()
-    x = self.norm[2](self.conv[2](x) * mult).relu()
+    mult = self.se2(self.se1(residual.mean((2,3))).clip(1e-15, 1e15).gelu()).sigmoid().reshape(x.shape[0], x.shape[1], 1, 1) if self.se else 1.0
```

Comment:
What's up with these weird clips?

Reply:
gelu causes NaNs if there are no clips, probably gradidnt clipping would be a better idea

Reply:
Then fix it in gelu, not here